Sources/LiveViewNative/Utils/ShapeStyle/ShadowStyle.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  ShadowStyle.swift
+//  
+//
+//  Created by Carson Katri on 4/25/23.
+//
+
+import SwiftUI
+
+/// A shadow configuration.
+///
+/// Two types of shadow can be created, `:drop` and `:inner`. Use a tuple where the first element is the type and the second is a keyword list or map with any options.
+///
+/// The `radius` argument is required.
+///
+/// ```elixir
+/// {:drop, [radius: 10]}
```

Comment:
@carson-katri Hmm, I tried this but it doesn't seem to be working:

```heex
<VStack id="hello-ios" modifiers={@native |> foreground_style(primary: {:shadow, {:drop, [radius: 10]}})}>
  <Spacer />
  <HStack modifiers={@native |> padding(all: 5)}>
    <Text>Hello iOS!</Text>
  </HStack>
</VStack>
```

![Screenshot 2023-04-25 at 1 21 08 PM](https://user-images.githubusercontent.com/5893007/234394730-b165ba0a-410b-4052-b707-62ce439d7ece.png)

Reply:
Shadow is a modifier that applies to another shape style, so you would need something like:
```elixir
{:foreground, [{:shadow, {:drop, [radius: 10]}}]}
```